index.html

```diff
@@ -27,5 +27,7 @@
       <input class="deposit" type="button" value="Deposit" />
       <input class="withdraw" type="button" value="Withdraw" />
     </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
+    <script src="atm.js"></script>
```

Comment:
You link to `atm.js` twice in this file. This one is correct -- the one in `<head>` is being loaded prior to jQuery, which is why you might be seeing `Uncaught ReferenceError: $ is not defined` in your browser console.